content/variables/using-environment-variables.md

```diff
@@ -29,6 +29,28 @@ Add-Content -Path $env:CM_ENV -Value "KEY=value"
 
 You can then reference the variable in subsequent parts of your workflow by using `$env:KEY`.
 
+### Example usage
+
+The variables that you write to the `CM_ENV` environment file work with both `codemagic.yaml` and `Flutter Workflow Editor` configurations. Note that the examples below are specific to Linux and macOS build machines. On Windows machines, the correct way to reference an environment variable is using `$env:KEY`.
+
+#### codemagic.yaml
+
+```yaml
+scripts:
+  - name: Set the value
+    script: |
+      echo "KEY=value" >> $CM_ENV
+  - name: Use the value
+    script: |
+      echo $CM_ENV # This will output 'value'
```

Comment:
not correct example

but let's revert this commit - currently the examples are too abstract and don't look helpful just add more complexity